energy-integration/fees-collector/tests/fees_collector_rust_test.rs

```diff
@@ -1594,20 +1442,212 @@ fn redistribute_rewards_test() {
                 // Rewards were put in current_week storage (i.e. 10)
 
                 let first_token_balance = sc
-                    .accumulated_fees(10, &managed_token_id!(FIRST_TOKEN_ID))
-                    .get();
-                let second_token_balance = sc
-                    .accumulated_fees(10, &managed_token_id!(SECOND_TOKEN_ID))
+                    .accumulated_fees(10, &managed_token_id!(BASE_ASSET_TOKEN_ID))
                     .get();
 
                 // i.e. 6 weeks worth of rewards minus what the third user claimed
                 assert_eq!(
                     first_token_balance,
                     managed_biguint!(599_952_417_140_485_515u64)
                 );
+            },
+        )
+        .assert_ok();
```

Comment:
Also check the storage is properly cleared for the past weeks, after the rewards redistribution.

Reply:
Also, call `redistribute_rewards`  to see what happens.

Reply:
Done.